ocfweb/reservations/reserve.py

```diff
@@ -0,0 +1,214 @@
+import email.mime.text
+import subprocess
+from collections import namedtuple
+from email.utils import parseaddr
+
+import ocflib.misc.validators
+from django import forms
+from django.shortcuts import render
+from jinja2 import Environment
+from jinja2 import PackageLoader
+from ocflib.account.search import user_attrs_ucb
+
+from ocfweb.auth import calnet_required
+from ocfweb.component.forms import wrap_validator
+
+SENDMAIL_PATH = '/usr/sbin/sendmail'
+MAIL_FROM = 'Open Computing Facility <help@ocf.berkeley.edu>'
```

Comment:
we shouldn't be redefining these, `ocflib.misc.mail` knows how to send email already

Reply:
Fixed